content/200-concepts/200-database-connectors/08-sql-server/index.mdx

```diff
@@ -106,10 +76,7 @@ model Artist {
 }
 ```
 
-If you want to use relations as described here, you must:
-
-- Create the migration using the `--create-only` flag
-- Modify the cascading rules manually before writing to the database
+See [Cyclic referential actions with Microsoft SQL Server](../../components/prisma-schema/relations/referential-actions/cyclic-referential-actions)] for information about how to fix this.
```

Comment:
I think there is no need to fix this any more, as we have validation which will already prevent it from being generated in a migration. The whole section here can probably be rewritten.

Reply:
Hmm, the section outlines what the problem is, shows you what error messages to expect then we link to the dedicated page which tells you how to solve said problem. Dont you think that enough?

Reply:
With the new validation implemented, you do not get these error message any more - as the validation already catches that before you create a SQL file to execute. So what error message SQL Server _would_ return is more background information that fits into the "Cyclic Referential Actions" page in the docs, but not on the SQL Server with Prisma page any more (As there we avoid this error message by having additional, fancy logic).

Reply:
I think you can make this _much_ shorter here, maybe just mention that "cyclic ref actions are a thing on SQL Server which leads to us having special handling and a nice error message, _go over hear to read more details_" and then make sure all the details are actually on that page and not here any more.